### ISSUE TITLE
man page error causes pager to not load

### ISSUE THREAD

**Issue by jrwrigh** [Jguer/yay#1157]
#### Affected Version
`yay v9.4.2 - libalpm v12.0.1`
`man` version: 2.9.0

#### Issue
When running `man yay`, the following error comes up:
`troff: <standard input>:1: name expected (got '\ '): treated as missing`

Possibly as a result, it does not open in a pager (ie. less).

#### Steps to reproduce
1. `\man yay` (note the `\man` is used to bypass an alias I have for man that colorizes the output)

#### Output
```
➜ \man yay
troff: <standard input>:1: name expected (got '\ '): treated as missing
YAY(8)                                             Yay Manual                                             YAY(8)

[.... the rest of the man page]
```

**Comment by Jguer** [comment]
I've tried to reproduce this (`man yay` , `\man yay` (which works even without an alias and `man yay | less` to remove color) and Im unable to reproduce this issue

**Comment by jrwrigh** [comment]
What version of man are you using? Could be a version issue on my side(?).

**Comment by Jguer** [comment]
yay 9.4.4
man 2.9.0
less 551
shell: zsh

**Comment by jrwrigh** [comment]
Thanks. I should've checked, but something is up with `man` for me. I don't get the extra `troff` error, but it's not launching the output inside `less`.

**Comment by jrwrigh** [comment]
For any other people with this issue (both `troff` error and `man` not using `less`), see my [StackExchange question/answer here](https://unix.stackexchange.com/a/562814/297140). 

tl;dr. I had a `zsh` plugin that "overwrote" `env` to use `grc`  ([general colourizer](https://github.com/garabik/grc)), which caused a different plugin that used `env` for redefining `man` to fail.
